### PR TITLE
Host entire email subscription process

### PR DESCRIPTION
**Pull request by clementbiron**

### CODE REVIEW

**Comment by MattiSG** [comment]
I feel that the first merge commits should not be part of this PR.

File: src/translations/fr/dashboard.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   "latestversionscommits.buttonblock.label": "Explorer",
   "latestversionscommits.buttonblock.link.title": "Explorer les versions de documents",
   "latestversionscommits.buttonblock.title": "Nous avons enregistré {{count}} versions de documents",
+  "latestversionscommits.buttonblock.title_plural": "__STRING_NOT_TRANSLATED__",
```
Comment (MattiSG) [inline src/translations/fr/dashboard.json:16]: Why is this added in this PR? This should in any case be translated.

File: src/translations/fr/subscribe/confirm.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Presque terminé!
```
Comment (MattiSG) [inline src/translations/fr/subscribe/confirm.mdx:1]: (non-breakable) spaces before `?`, `!`, `:` and `;` in French 🙂 
```suggestion
# Presque terminé !
```

File: src/translations/fr/subscribe/confirm.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "seo.title": "Abonnement presque terminé !"
```
Comment (MattiSG) [inline src/translations/fr/subscribe/confirm.json:2]:
```suggestion
  "seo.title": "Confirmez votre abonnement"
```

File: src/translations/en/subscribe/confirm.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "seo.title": "Thanks for subscribing"
```
Comment (MattiSG) [inline src/translations/en/subscribe/confirm.json:2]:
```suggestion
  "seo.title": "Confirm your subscription"
```

File: src/translations/fr/subscribe/confirm.mdx [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+**Vous devriez recevoir un e-mail avec un lien de confirmation.**
```
Comment (MattiSG) [inline src/translations/fr/subscribe/confirm.mdx:3]: Please use consistent syntax in the whole text: either “e-mail” or “email” (or, in French, “courriel” would be welcome  🙂).

File: src/translations/fr/subscribe/confirm.mdx [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+Pour protéger les destinataires d'emails non désirés, nous nous assurons que l'adresse email que vous avez donnée souhaite être abonnée. Cliquez sur le lien dans l'email que nous venons de vous envoyer pour confirmer l'abonnement 🙂.
```
Comment (MattiSG) [inline src/translations/fr/subscribe/confirm.mdx:5]: Not sure about the punctuation after the emoji.
```suggestion
Pour protéger les destinataires d'emails non désirés, nous nous assurons que l'adresse email que vous avez donnée souhaite être abonnée. Cliquez sur le lien dans l'email que nous venons de vous envoyer pour confirmer l'abonnement 🙂
```

File: src/translations/fr/subscribe/done.mdx [hunk 7]
```diff
@@ -0,0 +1,5 @@
+# Vous êtes maintenant inscrit !
```
Comment (MattiSG) [inline src/translations/fr/subscribe/done.mdx:1]: Gender neutrality.

```suggestion
# Votre inscription est confirmée !
```

File: src/translations/fr/subscribe/done.mdx [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+
+**Nous vous enverrons les prochaines mises à jour dès qu'elles arriveront.**
```
Comment (MattiSG) [inline src/translations/fr/subscribe/done.mdx:3]:
```suggestion
**Nous vous enverrons les prochains mémos dès qu'ils seront prêts.**
```

File: src/translations/fr/subscribe/done.mdx [hunk 9]
```diff
@@ -0,0 +1,5 @@
...
+
+Vos commentaires sont toujours les bienvenus, n'hésitez à nous dire comment nous pourrions les e-mails 🙂
```
Comment (MattiSG) [inline src/translations/fr/subscribe/done.mdx:5]: Quite a few words missing in this sentence 😉 

```suggestion
Vos commentaires sont toujours les bienvenus, n'hésitez pas à nous dire comment nous pourrions améliorer les mémos 🙂
```

But let's rephrase in a more precise way:

```suggestion
Nous avons besoins de vos retours : répondez à n’importe lequel de nos mémos pour nous dire comment nous pourrions les améliorer 🙂
```

File: src/translations/en/subscribe/confirm.mdx [hunk 10]
```diff
@@ -0,0 +1,7 @@
+# Almost done!
+
+**You should receive an email with a confirmation link.**
```
Comment (MattiSG) [inline src/translations/en/subscribe/confirm.mdx:3]: Isn't this a bit small compared to the main title? 🤔 

```suggestion
## You should receive an email with a confirmation link.
```

Screen Shot 2022-06-12 at 22 38 29

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: src/translations/en/subscribe/confirm.mdx (hunk 10 above)
Comment (clementbiron) [reply src/translations/en/subscribe/confirm.mdx:3]: Yes, i corrected.

File: src/translations/fr/dashboard.json (hunk 1 above)
Comment (clementbiron) [reply src/translations/fr/dashboard.json:16]: Fixed here https://github.com/ambanum/OpenTermsArchive.org/pull/131/commits/e5ba03ea32c26116cf50577b72027600ac7a799b

**Review by MattiSG — APPROVED** [review]
